client/js/components/statement/listStatements/ListStatements.vue

```diff
@@ -231,7 +231,7 @@
           </statement-meta-data>
 
           <!-- Statement text -->
-          <div class="u-pt-0_5 break-words">
+          <div class="u-pt-0_5 break-words table-text">
```

Comment:
@spiess-demos do we have or want to have a generic reset-class?

Reply:
As i understand the code changes, it is not about a reset, but about the same issue that has arisen in different places since we employed the tailwind reset for ul and ol elements.

for this, i'd argue to use `c-styled-html`  which is defined here: DemosPlanCoreBundle/Resources/client/scss/components/_styled-html.scss:86 

If you like, you could extend it with the p reset:

``` 
           & li > p {
                margin: 0;
            }
```

However i tend to believe that the actual bug originates in the splitStatement logic which does not wrap li elements within subsequent segments with the needed `ul` elements.

Reply:
The problem is that the `ul` and `ol` elements have a global `list-style: none` declaration. To resolve this, we need to specify the list style where it's required...

If you suggest that `c-styled-html` is the appropriate class for these purposes, I have to make some adjustments 
(like add the list style) --> https://github.com/demos-europe/demosplan-core/pull/1961/commits/f9f46c2452acb7b8bdb95d564b4f98095eae3358 Is it ok?

Does the `li` tag really require the `margin-bottom: 12px` as declared in the `c-styled-html` class?
![image](https://github.com/demos-europe/demosplan-core/assets/110987766/d99aa2ee-5b41-4dab-86af-353e07a85dd4)

or can it be removed? 

![image](https://github.com/demos-europe/demosplan-core/assets/110987766/569a1eb4-16e6-4287-8aa8-4e58331fc42e)

Reply:
It all goes back to [this commit of mine](https://github.com/demos-europe/demosplan/commit/8c1f5b5a55b8e221e19b9b182b72d5aa5c939877). We can see the importance of good commit messages here :) However i remember that _paragraph.scss is the precursor of _styled-html.scss, and it was first bootstrapped to style the content in templates/bundles/DemosPlanCoreBundle/DemosPlanDocument/public_paragraph_document.html.twig:183. However this was just as opinionated as it would probably be to remove the margin-bottom, i guess it is ok to remove it!